man/bundle-config.ronn

```diff
@@ -155,6 +155,14 @@ learn more about their operation in [bundle install(1)][bundle-install].
    The number of redirects allowed for network requests. Defaults to `5`.
 * `timeout` (`BUNDLE_TIMEOUT`):
    The seconds allowed before timing out for network requests. Defaults to `10`.
+* `force_ruby_platform` (`BUNDLE_FORCE_RUBY_PLATFORM`):
+   Force bundler to install the `ruby` platform version of gems, instead of
+   platform-specific gems.
+* `specific_platform` (`BUNDLE_SPECIFIC_PLATFORM`):
+   Allow bundler to resolve for the specific running platform and store it in
+   the lockfile, instead of only using a generic platform.
+* `disable_checksum_validation` (`BUNDLE_DISABLE_CHECKSUM_VALIDATION`):
+   Ignore the checksums of downloaded gems returned by rubygems sources.
```

Comment:
Is the only practical effect of this setting to allow installing gems even if they do not match the checksum provided by RubyGems? If so, let's say that instead. We want to document the results of the settings whenever we can. 👍